pkg/virtctl/imageupload/imageupload.go

```diff
@@ -82,6 +82,8 @@ var (
 	storageClass   string
 	imagePath      string
 	accessMode     string
+	volumeMode     string
+	volMode        string
```

Comment:
I have no idea what is the difference between these to variables. The name does not help me.

Reply:
The access mode line is in my forgotten zone, will revert it.